src/Client.js

```diff
@@ -137,9 +137,12 @@ class Client extends EventEmitter {
      * @param {string} message 
      */
     async sendMessage(chatId, message) {
-        await this.pupPage.evaluate((chatId, message) => {
-            Store.SendMessage(Store.Chat.get(chatId), message);
-        }, chatId, message)
+        let last_message = await this.pupPage.evaluate(async (chatId, message) => {
+            await Store.SendMessage(Store.Chat.get(chatId), message);
+            return Store.Chat.get(chatId).msgs._last.serialize()
```

Comment:
This is actually a pretty nice idea, however, this implementation is error prone since you don't have any guarantees that the last message in the chat is the message you sent. 

I was able to break this with the example in the repo by putting my other phone in airplane mode, sending `!ping` along with a couple more messages, and then turning airplane mode off. All the messages arrive to our client seemingly at the same time, but in between the time it takes to send a reply to the `!ping` command and when you save that `last_message`, another one of the messages that were coming in was received, so the message sent by the client is in fact *not* the last message in the chat.

Reply:
Thanks for the fast reply. Will work on the other stuff after the weekend and plan on adding a lot more. Any idea on how it would be possible to successfully retrieve the sent message?because returning Store.SendMessage only outputs the string "success".

Reply:
@pedroslopez Pinging you in case you didn't see it. Btw. do you use discord by any chance?

Reply:
Hey! Hadn't seen this. I'm not sure to be honest how we would be able to reliably get the latest message. Maybe it can be done matching the last message based on the content itself and things like user? Not sure.

Yeah I'm on discord, nacho#6294 if you want to message me.

Reply:
I'm still not convinced on this issue... maybe leave this out until we find a more definitive way to get the sent message?